test.sh

```diff
@@ -78,17 +78,20 @@ function run {
     DATA_TRANSPORT_LAYER_TAG=$DATA_TRANSPORT_LAYER_TAG \
         $cmd > $artifacts_folder/process.log # Send all process logs to process.log
```

Comment:
Probably not a bad idea to redirect stderr to the file here too with `2>&1`
What do you think about using `tee` here? Then it will still print out and not need another tab for `tail -f` or something

Reply:
I think of these suggestions make sense, I'll address in the base PR.